lib/sequent/core/ext/ext.rb

```diff
@@ -0,0 +1,118 @@
+class Symbol
```

Comment:
Is it possible to avoid monkey-patching core classes? It's not exactly gem best-practice.